packages/node_modules/@node-red/editor-client/src/js/ui/palette-editor.js

```diff
@@ -607,6 +609,10 @@ RED.palette.editor = (function() {
         updateDenyList = RED.utils.parseModuleList(updateDenyList);
         updateAllowed = RED.settings.get("externalModules.palette.allowUpdate",true);
 
+        if (RED.settings.get("editorTheme.palette.checkForUpdates", true) === true) {
```

Comment:
Seeing as you are adding a setting - why not make this the actual interval instead of just true/false... If set to a vlid number then use it - if not then no update check.

Reply:
Initially it was a minimum interval of 30min. Nick requested for this first iteration to be a boolean. See [Slack](https://node-red.slack.com/archives/C9RSU6S56/p1730218556745499?thread_ts=1730211021.585789)

Reply:
🤷 OK - as per other discussions I'm not convinced it's needed at all as the main catalogue only updates every 30 mins or so - if you need to force a reload manually to check for a node you know has updated it's not hard to reload the editor.